### PR TITLE
Make rotation via construction emit effect & sound

### PR DESCRIPTION
**Pull request by Quezler**
currently when you build over something with a different rotation it just shows the beams and it happening, but no visual ques.

this pull makes it so that when its being rotated via construction that it shows the construction effect & sound.
(rotation via the mouse wheel is exempt)

this was the cleanest place to add the sound hooks in (due to shouldplay and pitch being private), instead of moving the entire function over here, and on first glance it might look as though scroll wheel rotate triggers this too, which it does not.

in short, this will make it feel better when you rotate lots of stuff like a bunch of conveyors in bulk.

### CODE REVIEW

**Comment by Anuken** [comment]
Rotating with E+Scrollwheel doesn't shown any visual effects either. I don't think it should play a sound or make an effect here.

**Comment by Quezler** [comment]
it just feels "wrong" to build a conveyor partially over another conveyor and not hear or see anything just because it only rotates

**Comment by XEN0PHIL** [comment]
If something is easy to do and it will make players happy without affecting anything in a bad way then why not?
Building something should feel like it's being built.

**Comment by Anuken** [comment]
In this case, it's *not* being built. The block is not being re-created. It should *not* look like a standard building being placed.

**Comment by XEN0PHIL** [comment]
I mean it should make you feel like you changed something. I'd even add an effect of rotating something for the same feeling. And as an easier detection of stealthy block rotation griefing

**Comment by XEN0PHIL** [comment]
Perhaps the block placement sound but without the building process humming?

**Comment by grannybullet** [comment]
Yeah this would be a nice change, it does feel a little dull just flipping the orreintation of a block with no feedback.

**Comment by Anuken** [comment]
If an effect is added, it should happen with both types of rotation, and not look identical to the block place animation.

**Comment by Quezler** [comment]
Made obsolete by 69eb8faf67e49d83f4b3c18278937f373c08b012
